media/img/firefox/new/horizon/down-arrow.svg

```diff
@@ -0,0 +1,14 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 100 100">
+  <path
```

Comment:
You may have done this intentionally, but this svg could be minified to save on whitespece?